modules/core/02-client/keeper/keeper.go

```diff
@@ -413,3 +413,23 @@ func (k Keeper) GetClientStatus(ctx sdk.Context, clientState exported.ClientStat
 	}
 	return clientState.Status(ctx, k.ClientStore(ctx, clientID), k.cdc)
 }
+
+// GetParams returns the total set of ibc-client parameters.
+func (k Keeper) GetParams(ctx sdk.Context) types.Params {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get([]byte(types.ParamsKey))
+	if bz == nil {
```

Comment:
mega nit:

```suggestion
	if len(bz) == 0 {
```

Reply:
In the other PRs, we decided to use `bz == nil` since we also decided to panic. But I suppose this is ok here